package.json

```diff
@@ -62,7 +62,7 @@
     "multer": "^1.0.1",
     "node-uuid": "~1.4.3",
     "npm-run-all": "^1.5.1",
-    "nunjucks": "^2.3.0",
+    "nunjucks": "2.4.3",
```

Comment:
Hm, `^2.3.0` means "any version 2.3.0 or higher, as long as the major version number stays the same", so this should have already found 2.4.3 when doing a new `npm install` - Due to how semantic versioning works, where the first number is a "major release", the second number is the "minor" release, and the third number is a patching number, anything has the same major version should be API compatible, so we should be able to say `^2.4.3`, keeping the caret because newer versions in the `2.*.*` range should be compatible with the calls we make to nunjucks.